### ISSUE TITLE
iPad low memory crash multiple iossliders

### ISSUE THREAD

**Issue by ericbeekman** [iosscripts/iosslider#276]
Hi,

We are experiencing issues with multiple iOS sliders (4) on 1 page.

We have done this before without any problems, but this one keeps crashing. Even when we lazy load the images. When disabling the iossliders it runs great, so we really narrowed it down to that.

Has anyone have any idea how to fix it?

http://www.asbreda.nl/index.php (don't forget the index.php, since it's not public live yet)

Thanks for any suggestions.

**Comment by ericbeekman** [comment]
We fixed this one by lazy loading and destroying iOS sliders and images. 

If anyone also experiences this, we used this plugin for listening for appearing of images and iOS sliders: https://github.com/morr/jquery.appear

**Comment by t0rpedo** [comment]
@ericbeekman Thanks for that solution!

**Comment by levipadre** [comment]
Hi @ericbeekman,
Can you recall how did you managed the lazy loading problem? I tried jquery.appear, but nothing happend:

```
function slideLoaded() {
$(document.body).on('appear', '.iosSlider .item', function(e, $affected) {
    console.log('slide');
  });
}
```

Can you help me a bit?

**Comment by ericbeekman** [comment]
Hi @levipadre,

I did use appear. For every slider container and for every slide in the slider. 

The loading of the initial iosSlider on appear and ondissapear destroying it as well, but remembering the active slide for a possible reinit.

When it was loaded I also used lazy loading on the images on slides. The slides are div elements containing a data-src attribute with the src URL. On appear I created the img element.

<div class="photo lazyload" data-src="/files/cache/image.JPG"></div>

And I also removed the img elements as well. 

Combined it saved a lot. Without removing and destroying it crashed eventually.
